atomics/include/desul/atomics/Fetch_Op_Generic.hpp

```diff
@@ -86,10 +86,13 @@ DESUL_IMPL_ATOMIC_FETCH_OP_SHIFT_HOST_AND_DEVICE(rshift)
         load_operator<T, const T>(), const_cast<T*>(dest), T(), order, scope); \
   }                                                                            \
                                                                                \
+  // NOTE: using atomic_oper_fetch in the fallback implementation of atomic_store
+  // to avoid reading potentially uninitialized values which would yield
+  // undefinied behavior.
```

Comment:
```suggestion
// NOTE: using atomic_oper_fetch in the fallback implementation of atomic_store
// to avoid reading potentially uninitialized values which would yield
// undefined behavior.
```
Sorry I was silly and indented the code I had written.  Also fixing the typo here.